library-core/src/main/java/com/mikepenz/aboutlibraries/Libs.kt

```diff
@@ -82,21 +89,44 @@ public class Libs(context: Context, fields: Array<String> = context.getFields())
                 licenses.add(license)
             }
         }
-        //add internal libs
-        for (internalIdentifier in foundInternalLibraryIdentifiers) {
-            val library = genLibrary(context, internalIdentifier)
-            if (library != null) {
-                library.isInternal = true
-                internLibraries.add(library)
-            }
-        }
 
-        //add external libs
-        for (externalIdentifier in foundExternalLibraryIdentifiers) {
-            val library = genLibrary(context, externalIdentifier)
+        //add plugin libs
+        for (pluginLibraryIdentifier in foundPluginLibraryIdentifiers) {
```

Comment:
This is a matter of personal preference, but there are several ways to write this in a potentially more kotlinized way.

The use of null checks with if statements, or even `T?.let { t -> }` results in a lot of nested braces.
For this one, you can consider adding a separate function (inner function if required) of the form:

```kt
fun someFunctionName(libIdentifier: Type): Library? {
   val library = genLibrary(context, libIdentifier) ?: return null
   ...
   val enchantWithKey = ... ?: return library
   val enchantWith = ... ?: return library
   library.enchantBy(enchantWith)
   return library
}
```

and then do something like

```kt
externLibraries = foundPluginLibraryIdentifiers.mapNotNull { someFunctionName(it) }
```

(or `addAll` if you require mutation)